OpenRA.Mods.Common/Widgets/LabelWidget.cs

```diff
@@ -28,15 +30,42 @@ public class LabelWidget : Widget
 		public bool Contrast = ChromeMetrics.Get<bool>("TextContrast");
 		public Color ContrastColor = ChromeMetrics.Get<Color>("TextContrastColor");
 		public bool WordWrap = false;
+
+		[Desc("Space between lines as a percentage (default) of line height or fixed pixel amount (rounded down).")]
+		public int LineSpacing = 40;
```

Comment:
My opinion is that we should use integer percentage everywhere possible for consistency.